src/components/CentralContainer/Sidebar/SidebarTabs/index.tsx

```diff
@@ -63,6 +66,9 @@ const SidebarTabs = forwardRef<HTMLDivElement, SidebarTabsProps>((
             case TAB_NAME.SETTINGS:
                 setIsSettingsModalOpen(true);
                 break;
+            case TAB_NAME.DOCUMENTATION:
+                window.open(DOCUMENTATION_URL, "_blank");
```

Comment:
```suggestion
                window.open(DOCUMENTATION_URL, "_blank", "noopener");
```

Unless it's not necessary in this case?

Reply:
Right, I was thinking omitting noreferrer makes sense here since we're linking to our own docs.